### PR TITLE
Make flowgen support sequence numbers in TCP.

### PR DESCRIPTION
**Pull request by justinemarie**

### CODE REVIEW

File: core/modules/flowgen.cc [hunk 1]
```diff
@@ -340,7 +346,7 @@ void FlowGen::GeneratePackets(bess::PacketBatch *batch) {
     uint64_t t;
     struct flow *f;
     bess::Packet *pkt;
-
+
```
Comment (melvinw) [inline core/modules/flowgen.cc]: nite: unnecessary indentation

File: core/modules/flowgen.cc [hunk 2]
```diff
@@ -85,6 +86,7 @@ inline struct flow *FlowGen::ScheduleFlow(uint64_t time_ns) {
   flows_free_.pop_front();
 
   f->first = 1;
+  f->next_seq_no = 12345;
```
Comment (melvinw) [inline core/modules/flowgen.cc]: ...why this value?

File: core/modules/flowgen.cc [hunk 3]
```diff
@@ -145,6 +147,7 @@ void FlowGen::PopulateInitialFlows() {
 
       /* overwrite with a emulated pre-existing flow */
       f->first = 0;
+      f->next_seq_no = 56789;
```
Comment (melvinw) [inline core/modules/flowgen.cc]: same here

File: core/modules/flowgen.cc (hunk 2 above)
Comment (justinemarie) [inline core/modules/flowgen.cc]: It's recognizable for debugging purposes.

File: core/modules/flowgen.cc (hunk 3 above)
Comment (justinemarie) [inline core/modules/flowgen.cc]: Also same -- these packets here are part of the "initial" backfilled flows.

File: core/modules/flowgen.cc (hunk 2 above)
Comment (melvinw) [inline core/modules/flowgen.cc]: kk

**Pull request merged**